src/lib/services/email-service.ts

```diff
@@ -173,7 +173,9 @@ export class EmailService {
                 ? `${this.config.server.unleashUrl}/projects/${project}/archive?sort=archivedAt&search=${flagName}`
                 : false;
 
-            const conflictingChangeRequestLink = `${this.config.server.unleashUrl}/projects/${project}/change-requests/${conflictingChangeRequestId}`;
+            const conflictingChangeRequestLink = conflictingChangeRequestId
+                ? `${this.config.server.unleashUrl}/projects/${project}/change-requests/${conflictingChangeRequestId}`
+                : false;
```

Comment:
maybe null instead to avoid string | boolean type?

Reply:
This is needed for the mustashe template conditional check

Reply:
The tags in mustashe will not render when `false` . Not sure if null will act the same, I think it doesn't